webpack/redux/reducers/RedHatRepositories/enabled.js

```diff
@@ -82,7 +83,9 @@ export default (state = initialState, action) => {
           // server can return per_page: null when there's error in the search query,
           // don't store it in such case
           // eslint-disable-next-line camelcase
-          perPage: Number(per_page || state.pagination.perPage),
+          perPage: (per_page || state.pagination.perPage)
+          // eslint-disable-next-line camelcase
+            && Number(per_page || state.pagination.perPage),
```

Comment:
So I remember you mentioning that sending NaN as per_page causes the issue while sending undefined does not.
Verified that Number(undefined || undefined) returns NaN whereas (undefined || undefined) && Number(undefined || undefined) returns undefined.

Reply:
correct.